JuliaInterface/src/convert.c

```diff
@@ -25,6 +25,14 @@ jl_value_t * julia_gap(Obj obj)
     if (obj == False) {
         return jl_false;
     }
+    if (TNUM_OBJ(obj) == T_COMOBJ &&
+        CALL_1ARGS(JULIAINTERFACE_IsTransparentJuliaWrapper, obj) == True) {
+        Obj return_value = CALL_1ARGS(JULIAINTERFACE_JuliaPointer, obj);
+        if (!IS_JULIA_OBJ(return_value)) {
+            ErrorMayQuit("JuliaPointer is not a Julia object", 0, 0);
```

Comment:
```suggestion
            ErrorMayQuit("JuliaPointer must be a Julia object (not a %s)", (int)TNAM_OBJ(return_value), 0);
```
That would fit better with the usual style of error messages (at least as used in the GAP kernel)